test/no-send.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @fileoverview Tests for the no-send rule
+ * @author Tristan Homsi <tristanhomsi@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require("solium");
+
+var userConfig = {
+	rules: {
+		"security/no-send": "error"
+	}
+};
+
+describe ('[RULE] no-send', function () {
```

Comment:
please change description to `[RULE] no-send: Rejections` (this is convention throughout Solium)